packages/gestalt/src/Video.js

```diff
@@ -20,7 +20,6 @@ type Props = {|
   accessibilityUnmuteLabel: string,
   aspectRatio: number,
   captions: string,
-  children?: React.Node,
```

Comment:
@jesselumarie what's the reason for removing this line? Could we add it back in?

![image](https://user-images.githubusercontent.com/127199/87199832-24768080-c2b0-11ea-96d0-11fb9fefa7b0.png)

Reply:
definite rebase mistake -- good catch